record-builder-processor/src/main/java/io/soabase/recordbuilder/processor/InternalRecordBuilderProcessor.java

```diff
@@ -571,35 +619,7 @@ public String p2() {
             });
             witherClassNameBuilder.add(">");
         }
-        var witherClassName = witherClassNameBuilder.build().toString();
-        var codeBuilder = CodeBlock.builder()
-                .add("return new $L", witherClassName)
```

Comment:
Jacoco does not check anonymous class annotations, so changing to `new @RecordBuilderGenerated With() {}` did not exclude the inner class from coverage